packages/cozy-device-helper/package.json

```diff
@@ -30,9 +31,12 @@
     "url": "git+https://github.com/cozy/cozy-libs.git"
   },
   "scripts": {
-    "build": "babel src -d dist --ignore *.spec.js --copy-files",
+    "build": "yarn clean && yarn types && babel --extensions .ts,.tsx --ignore '**/*.spec.tsx','**/*.spec.ts' ./src -d ./dist",
+    "clean": "rm -rf ./dist",
     "prepublishOnly": "yarn run build",
-    "test": "jest src/**"
+    "start": "yarn build --watch",
+    "test": "jest src/**",
+    "types": "tsc -p tsconfig-build.json"
   },
-  "types": "index.d.ts"
+  "types": "dist/index.d.ts"
```

Comment:
isn't it a refactor commit?

Reply:
yes the location changed after the refactor

Reply:
I think we should rename this commit:
"refactor: Convert cozy-device-helper to Typescript"